sp800-63c/sec6_presentation.md

```diff
@@ -2,10 +2,12 @@
 
 ## 6. Assertion Presentation
 
-Assertions MAY be presented in either a *back-channel* or *front-channel* manner from the IdP to the RP. Each model has its benefits and drawbacks, but both require the proper validation of the assertion. Assertions MAY also be proxied to facilitate federation between IdPs and RPs under specific circumstances, as discussed in section 4.1.4.
+Assertions MAY be presented in either a *back-channel* or *front-channel* manner from the IdP to the RP. There are tradeoffs with each model, but both require the proper validation of the assertion. Assertions MAY also be proxied to facilitate federation between IdPs and RPs under specific circumstances, as discussed in section 4.1.4.
 
 The IdP SHALL transmit only those attributes that were explicitly requested by the RP. RPs SHALL conduct a privacy risk assessment when determining which attributes to request. 
 
+>**MG: Too much to require the risk assessment be made avaiable to the IdP or proxy?**
```

Comment:
Yes, definitely too much to ask, especially when you consider dynamic relationships. We'd need to get into the business of specifying what providing that assessment means, and also have to have guidelines about what information is included in such an assessment as they may contain sensitive system configuration information by default.